README.md

```diff
@@ -52,34 +60,49 @@ Options:
 
 ## annotated spec:
 
+The below example can be compiled by running `cogs gen basic examples/basic.cog.toml`.
```

Comment:
can we point the tests in `.github/workflows/test.yaml` to the new file locations?